privacy-snapshot.json

```diff
@@ -27,6 +27,7 @@
   "etherscan.io",
   "execution.metamask.io",
   "fonts.gstatic.com",
+  "foo.io",
```

Comment:
🤔

Reply:
would require updating api-specs to replace the example params with something else